### DISCUSSION TITLE
pyboard v1.1 bricked?

### DISCUSSION THREAD

**Discussion by joh02** [micropython/micropython#9419] [Hardware & Peripherals] [answered]
Hi,
after longer normal work with my pyboard v1.1 is now its filesystem in Linux only r/o mounted. The files main.py and boot.py are visible, but seem to be filled only with 0xff. But: The interpreter reports normally via a minicom connection and accepts instructions:

`RuntimeError: name too long
MicroPython v1.19.1 on 2022-06-18; PYBv1.1 with STM32F405RG
Type "help()" for more information.
>>>
`

I can't do a factory reset, after pressing reset and usr and releasing reset only you green LED is on, no matter how long I hold usr. How can I bring the board back to life?

greetings
joh

**Comment by jimmo** [comment, marked as answer]
@joh02  I think the best bet is to put it into DFU mode (see https://github.com/micropython/micropython/wiki/Pyboard-Firmware-Update ) and do a mass-erase, then re-flash the latest firmware.

**Reply by joh02** [reply]
thx, but: How can I do 'a mass-erase' ?
I couldn't find that in 
dfu-util --help

**Reply by peterhinch** [reply]
You can do it with [pydfu](https://github.com/micropython/micropython/blob/master/tools/pydfu.py):
```
$ ./pydfu.py --help
usage: pydfu.py [-h] [-l] [--vid VID] [--pid PID] [-m] [-u PATH] [-x] [-v]

DFU Python Util

optional arguments:
  -h, --help            show this help message and exit
  -l, --list            list available DFU devices
  --vid VID             USB Vendor ID
  --pid PID             USB Product ID
  -m, --mass-erase      mass erase device
  -u PATH, --upload PATH
                        read file from DFU device
  -x, --exit            Exit DFU
  -v, --verbose         increase output verbosity
```

**Reply by jimmo** [reply]
> thx, but: How can I do 'a mass-erase' ?

It's one of the options to `-s`.

**Comment by joh02** [comment]
after the mass-erase and flashing the threading image the pyboard is alive again. Thank you very much for the fast help.
